addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -252,7 +252,8 @@ def _get_invoice_line_item_vals(self, line, taxes_vals):
         for val in line_item_vals['classified_tax_category_vals']:
             # [UBL-CR-601] TaxExemptionReason must not appear in InvoiceLine Item ClassifiedTaxCategory
             # [BR-E-10] TaxExemptionReason must only appear in TaxTotal TaxSubtotal TaxCategory
-            val.pop('tax_exemption_reason')
+            if val.get('tax_exemption_reason'):
+                val.pop('tax_exemption_reason')
```

Comment:
Up to you to follow this suggestion or not :+1: 
```suggestion
            val.pop('tax_exemption_reason', None)
```

Reply:
:point_up: I also think this solution is more elegant :slightly_smiling_face:

Reply:
Thank you for the discussion! I committed the suggestion.